GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.cs

```diff
@@ -1,14 +1,26 @@
 ﻿using GitCommands;
+using ResourceManager;
 
 namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class DiffViewerSettingsPage : SettingsPageWithHeader
     {
+        private readonly TranslationString _showDiffForAllParents = new TranslationString(@"If set, show all differences (not just the first differences).
+
+For a single selected commit, show the difference to the parent commit.
```

Comment:
I don't think it is grammatically correct to say "_difference to_", I believe should be "_difference with_".

```suggestion
- For a single selected commit, show a difference between it and its parent commit.
```

Reply:
Correct, you use "different to" in BE and "different from/than" in AE though.